### PR TITLE
Misc docs

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

File: stream/stream.go [hunk 1]
```diff
@@ -34,7 +37,7 @@ type Stream struct {
 	Architectures map[string]Arch `json:"architectures"`
 }
 
-// Architecture release details
+// Arch is release details for a particular hardware architecture
```
Comment (lucab) [inline stream/stream.go:40]: s/is/contains/ maybe?

File: release/release.go [hunk 2]
```diff
@@ -1,3 +1,7 @@
+// Package release contains APIs for interacting with a
+// particular "release".  Avoid this unless you are sure
+// you need it.  In most cases it's expected
```
Comment (bgilbert) [inline release/release.go:3]: s/In most cases//.  So far we've successfully avoided users having to look here.

File: stream/stream.go [hunk 3]
```diff
@@ -1,3 +1,6 @@
+// Package stream models a CoreOS "stream", which is
+// a description of the binary images for CoreOS.   Use
```
Comment (bgilbert) [inline stream/stream.go:2]: Maybe "description of the recommended set of binary images"?

**Comment by cgwalters** [comment]
Rebased :surfer: and updated :new:

**Review by bgilbert — APPROVED** [review]

**Pull request merged**